src/test/java/com/puppycrawl/tools/checkstyle/utils/CommonUtilTest.java

```diff
@@ -576,6 +576,14 @@ public void testIsCodePointWhitespace() {
                 .isFalse();
     }
 
+    @Test
+    public void testIsCreatingInputSourceFromFilename()
+            throws CheckstyleException {
+        final String filename =
+            "/" + getPackageLocation() + "/InputCommonUtilTest_empty_checks.xml";
+        assertThat(CommonUtil.sourceFromFilename(filename)).isNotNull();
```

Comment:
we discourage all pure unit test as much as possible.
Ideally we should have no of them, if any is required due to inability to test by upper level of classes, we need a comment that explains reason in details.

what tool force you to have it ?

Reply:
I dont remember, I removed it, lets see if something fails